a4-speakup/templates/a4images/image_upload_widget.html

```diff
@@ -7,7 +7,7 @@
             {% if has_image_set and not is_required %}
                 <div class="input-group-append">
                     <label for="{{ checkbox_id }}" class="btn btn-danger" title="{% trans 'Remove the picture' %}">
-                        <i class="fas fa-trash-alt"></i>
+                        <i class="fas fa-trash-alt" aria-label="{% trans 'Remove the picture' %}"></i>
```

Comment:
will this not be read twice on a screen reader because of the title? should we use aria-labelled-by here? it still confuses me a bit!

Reply:
Yeah, I've read, that there should be both, but I think, chrome actaully ready both. So, don't know.
But in this file it is done like that with the upload right below. We should choose one of the options (1. add both, 2. only add aria-label or 3. only add title, but then aria-hidden to fa (last one seems dangerous to me)) we have and be consistent at least in the same file.